### ISSUE TITLE
Quartus 2 13.0sp1 Download Fails

### ISSUE THREAD

**Issue by cepawiel** [halfmanhalftaco/fpga-docker#5]
The curl command in Altera_Quartus_II_13.0sp1/Makefile currently just downloads a 0 byte file. Adding a -L flag to the command fixes it. I haven't checked the other versions of Quartus yet, but I'd assume they suffer from the same issue as the URLs look to be of the same format. If nobody gets to it before me I'll make a pull request with these changes.

**Comment by halfmanhalftaco** [comment]
Yeah, I think Intel started moving a lot of the download URLs, this could certainly use a refresh and a check to make sure download links still work.
